src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -87,6 +87,21 @@ public Commons getCommonsById(
     return ResponseEntity.ok().body(body);
   }
 
+  @ApiOperation(value = "Delete a commons from the table")
+  @PreAuthorize("hasRole('ROLE_ADMIN')")
+  @DeleteMapping(value = "/delete")
+  public ResponseEntity<String> deleteCommons(@ApiParam("commonsId") @RequestParam Long commonsId) {
+    log.info("id={}", commonsId);
+
+    // check if the commons is present in the table
+    commonsRepository.findById(commonsId)
+            .orElseThrow(() -> new EntityNotFoundException(Commons.class, commonsId));
+
+    commonsRepository.deleteById(commonsId);
+
+    return ResponseEntity.ok().body(String.format("commons with id %d deleted", commonsId));
+  }
```

Comment:
This is good... but it could be better.  I'll merge this, but I'll also put in an issue to improve this line.

Here's the issue: https://github.com/ucsb-cs156-w22/team04-w22-7pm-HappyCows/issues/61